server/session.go

```diff
@@ -630,6 +634,15 @@ func (s *Session) hello(msg *ClientComMessage) {
 	}
 	s.deviceID = msg.Hi.DeviceID
 	s.lang = msg.Hi.Lang
+	// Try to deduce the country from the locale.
+	if tag, err := language.Parse(s.lang); err == nil {
+		if region, conf := tag.Region(); region.IsCountry() && conf >= language.High {
+			s.countryCode = region.String()
+		}
+	}
```

Comment:
It may be useful to log if `s.lang` is longer than 2 chars but we are unable to extract the country. 
Longer than 2 chars because language.Parse("XX") gives confidence as 'Low'.

Reply:
Done.